packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -111,9 +118,34 @@ class BottomSheet extends StatefulWidget {
   /// If true, the bottom sheet can be dragged up and down and dismissed by
   /// swiping downwards.
   ///
+  /// This applies to the content below the drag handle, if showDragHandle is true.
+  ///
   /// Default is true.
   final bool enableDrag;
 
+  /// Specifies whether a drag handle is shown.
+  ///
+  /// The drag handle appears at the top of the bottom sheet. The default color is
+  /// [ColorScheme.onSurfaceVariant] with an opacity of 0.4 and can be customized
+  /// using dragHandleColor. The default size is Size(32,4) and can be customized
+  /// with dragHandleSize.
+  ///
+  /// Defaults to [BottomSheetThemeData.showDragHandle]. If that is also null,
```

Comment:
The wording about defaults for this property and the others is a little odd. Typical doc for this kind of thing would be: https://api.flutter.dev/flutter/material/Checkbox/checkColor.html. In this case that would be:
```suggestion
  /// If null, then the value of  [BottomSheetThemeData.showDragHandle] is used. If that is also null,
```